doc/services/dsp/index.rst

```diff
@@ -50,7 +50,9 @@ added to `subsys/dsp/Kconfig`_ along with the required dependencies and the
 ``default`` set for ``DSP_BACKEND`` Kconfig choice.
 
 Next, the implementation should be added at ``subsys/dsp/<backend>/`` and
-linked in at `subsys/dsp/CMakeLists.txt`_.
+linked in at `subsys/dsp/CMakeLists.txt`_. To add architecture-specific attributes,
+its corresponding Kconfig option should be added to `subsys/dsp/Kconfig`_ and use
+them to update ``DSP_DATA`` and ``DSP_STATIC_DATA`` in `include/zephyr/dsp/dsp.h`_.
```

Comment:
To fix the documentation build failure:

```suggestion
them to update ``DSP_DATA`` and ``DSP_STATIC_DATA`` in :file:`include/zephyr/dsp/dsp.h`.
```

p.s. the rest of this doc should be using `:file:` as well.